T/TempestRemap/build_tarballs.jl

```diff
@@ -72,11 +72,11 @@ products = [
 
 dependencies = [
     Dependency("OpenBLAS32_jll"),
-    Dependency("NetCDF_jll", compat="400.702.402 - 400.799"),
+    Dependency("NetCDF_jll", compat="400.702.402 - 400.902.5"),
     # The following is adapted from NetCDF_jll
     BuildDependency(PackageSpec(; name="MbedTLS_jll", version=v"2.24.0")),
 ]
 
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-    julia_compat="1.7",
+    julia_compat="1.8",
```

Comment:
Is this needed?

Reply:
🤷🏻‍♂️, I've never done this before, so probably not?

Reply:
Yeah, I don't think this is necessary at all